libraries/stdlib/stdlib_ng.mtlx

```diff
@@ -1160,6 +1160,143 @@
     <output name="out" type="float" nodename="N_range" />
   </nodegraph>
 
+  <!--
+    Node: <randomfloat>
+    Produces a randomized float, based on an 'input' signal and 'seed' value.
+  -->
+  <nodegraph name="NG_randomfloat_float" nodedef="ND_randomfloat_float">
+    <convert name="N_convertSeed1" type="float">
+      <input name="in" type="integer" interfacename="seed" />
```

Comment:
Isn't the `seed` input of his node already an integer?  (We really ought to catch this and emit a warning, but apparently we don't yet check for this case!)